src/pre_defined_classes/mathematical_program/medium/q8/MainTest.java

```diff
@@ -0,0 +1,30 @@
+package pre_defined_classes.mathematical_program.medium.q8;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    //Parsons
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The magnitude of an earthquake with intensity "),
+                new DoubleLiteral(23200.0, 23200.0, "intensity"),
```

Comment:
Is there a possibly different intensity line? If not you could just scrap the two 23200.0s. Either way, I'd say get rid of the name since you never use it

Reply:
No, but I just name everything for consistency

Reply:
Is this ok @Opeyem1a ?

Reply:
I think naming things like this is okay since it makes the code more readable, but it's not necessary for sure. I think when the double is a specific double though we should just catch any double (so no need to set a range)